package-lock.json

```diff
@@ -1347,7 +1347,7 @@
     },
     "figures": {
       "version": "2.0.0",
-      "resolved": "http://registry.npm.taobao.org/figures/download/figures-2.0.0.tgz",
```

Comment:
This  reminds me that I should switch my registry when creating the lock file...

Reply:
I think it got updated because i used `npm prune`.